Source/Menu/MainForm.cs

```diff
@@ -264,6 +262,31 @@ orderby tool.Text
             }
         }
 
+        private void LoadDocuments(List<ToolStripItem> docs, string folderPath, string code)
+        {
+            foreach (var filePath in Directory.GetFiles(folderPath))
+            {
+                var ext = System.IO.Path.GetExtension(filePath);
+                var name = System.IO.Path.GetFileNameWithoutExtension(filePath);
+                // These are the following formats that can be selected.
+                if (".pdf .doc .docx .pptx .txt".Contains(ext))
+                {
+                    var codeLabel = "";
+                    if (code != null)
+                        codeLabel = $" [{code}]";
+                    docs.Add(new ToolStripMenuItem($"{name}{ext}{codeLabel}", null, (Object sender2, EventArgs e2) =>
```

Comment:
```suggestion
                    docs.Add(new ToolStripMenuItem($"{name}{ext}{codeLabel}", null, (object sender2, EventArgs e2) =>
```
Visual Studio suggests using the lower-case name.